src/pages/SupplierInvoicePage.js

```diff
@@ -282,12 +436,37 @@ export class SupplierInvoicePage extends React.Component {
             }}
           />
         );
+      case FRIDGE_SELECT:
+        return (
+          <AutocompleteSelector
+            options={database.objects('Location')}
```

Comment:
i guess you are planning to change this (to make sure location is fridge) when the new modal with array input is available ?

Reply:
Yeah. From my PR comment. I should make them shorter :(

At the moment:

    Finalizing not in, will do in another (inventory adjustments etc)
    All locations are used in the modal. I will use GenericChooseModal once IC stock takes PR is merged into develop. (Can rebase vaccine-development?), but at the moment AutoCompleteSelector only takes a Realm.results also.
    I want to make a new component for the dosage entering modal. GenericNumberEntering or something which has validation and feedback for the user on the rare use case they try to enter non-numeric characters. Maybe some + and - buttons?